### PR TITLE
DOC-778: Stream worker ml topic

### PR DESCRIPTION
**Pull request by diana-macrometa**
This PR creates a topic to showcase our Streaming ML functions.

### CODE REVIEW

File: docs/cep/stream-worker-ml.md
Comment (StoyanTc) [inline docs/cep/stream-worker-ml.md]: @diana-macrometa - `OutputStream1` is an internal stream. Such streams are used for intermediate data processing and finally have to transfer its data to either `sink` or `store`. 
Might be `OutputStream1` should be replaced by `PredictionStream` preliminary defined as `sink` . Otherwise `PredictionStream` is not used at all.
it's the same within the other samples below.

**Review by StoyanTc — CHANGES_REQUESTED** [review]

File: docs/cep/stream-worker-ml.md
Comment (koshy-thomas) [inline docs/cep/stream-worker-ml.md]: @diana-macrometa we should also update this page (https://www.macrometa.com/docs/cep/query-guide/functions/streaming-ml/bayesianregression) The example in the above link uses the old QL.. and plus the example won't work directly it needs updateBayesianRegression just like you used it here.. so maybe we can update the example in the bayesianregression page to something similar to what you have added here..

**Review by elof — APPROVED** [review]
I approve this as far as it's not breaking anything, and the content looks good. I did not test to see if the stream workers work though

**Comment by diana-macrometa** [comment]
@elof, they passed the validator, but I like to wait for one or more of the SMEs to sign off before I merge. Especially because these are rather complex stream workers, and @koshy-thomas has occasionally been finding bugs in functions as he tests them.

File: docs/cep/query-guide/functions/streaming-ml/bayesianregression.md
Comment (koshy-thomas) [inline docs/cep/query-guide/functions/streaming-ml/bayesianregression.md]: We cannot use Example 1 directly we need to add somewhere that before using any bayesianRegression model we need to initialise it first using updateBayesianRegression... and that's why example2 is good.

File: docs/cep/query-guide/functions/streaming-ml/bayesianregression.md
Comment (koshy-thomas) [inline docs/cep/query-guide/functions/streaming-ml/bayesianregression.md]:
```suggestion
The `trainHousePriceModel` query processes events from the `HouseDataStream`, and builds or updates the Bayesian linear regression model 'housePriceModel' using the input features and the actual house prices.
```

File: docs/cep/stream-worker-ml.md
Comment (koshy-thomas) [inline docs/cep/stream-worker-ml.md]: @StoyanTc After validating this SW if you click on Validate again it says the regression model already exists, please use a different name but shouldn't it "simply update the regression model" and not throw the error??
Second point, shouldn't it create the model only when the SW is published? and not when it's validated? also, just some more scenarios for regression models to think of is if the regression model is being deleted if the SW is unpublished.. or if it resumes after SW crashes and restarts..

Comment (StoyanTc) [reply docs/cep/stream-worker-ml.md]: @koshy-thomas - We don't have experience with those extensions. 
Please file a ticket for investigating the protentional issue within.

Comment (diana-macrometa) [reply docs/cep/stream-worker-ml.md]: I ran into that, too. I had to keep renaming the models as I tested the SWs to try and get them valid.

File: docs/cep/query-guide/functions/streaming-ml/bayesianregression.md
Comment (diana-macrometa) [reply docs/cep/query-guide/functions/streaming-ml/bayesianregression.md]: I removed the Bayesian examples and hid the documentation until the bug Koshy is going to file gets worked out.

**Comment by diana-macrometa** [comment]
@StoyanTc, this is ready for your review.

File: docs/cep/stream-worker-ml.md
Comment (koshy-thomas) [reply docs/cep/stream-worker-ml.md]: @StoyanTc / @diana-macrometa : Bug filed :
https://macrometa.atlassian.net/browse/CEP-562

**Review by koshy-thomas — APPROVED** [review]

**Comment by diana-macrometa** [comment]
@StoyanTc, please review this PR

**Comment by diana-macrometa** [comment]
Reviewers, please review.

**Comment by diana-macrometa** [comment]
@StoyanTc, are you ready to unblock this?

**Review by StoyanTc — APPROVED** [review]

**Comment by StoyanTc** [comment]
> @StoyanTc, are you ready to unblock this?

Yes, @diana-macrometa

**Pull request merged**
